dmapiclient/search.py

```diff
@@ -116,7 +117,15 @@ def delete(self, index, service_id):
         return None
 
     def search_services(self, index, q=None, page=None, id_only=False, **filters):
-        response = self._get(self.get_search_url(index=index, q=q, page=page, id_only=id_only, **filters))
+        response = self._get(
+            self.get_search_url(index=index, doc_type='services', q=q, page=page, id_only=id_only, **filters)
+        )
+        return response
+
+    def search_briefs(self, index, q=None, page=None, id_only=False, **filters):
```

Comment:
The aggregate function (only one, you pass in the type as a parameter) is now inconsistent with the search functions (there are two, the type is based on which function you call).

I think I prefer just having the one function in the library (i.e. maybe we could have "search" rather than "search_services" and "search_briefs".

And do we not need an 'iter' version of the briefs search?

Reply:
Yeah, this makes sense. I think I was keeping them separate for backwards compatibility, but it should be fine.

And no, no need for a briefs iter.